charts/app/Chart.yaml

```diff
@@ -3,3 +3,8 @@ name: app
 description: A Helm chart for Kubernetes for running an application
 type: application
 version: 3.0.0
+
+metadata:
```

Comment:
Metadata is not a valid field in Chart.yaml so this most likely wont do anything https://helm.sh/docs/topics/charts/#the-chartyaml-file. What are we actually trying to achieve with the tagging here?

Reply:
I sent Adam off to try to run this. Hoping the same conclusion :D

Reply:
damn, made a mistake

Reply:
The actual goal is to annotate Kubernetes resources with an application_service and a spoc.

Okay, so now that I've pulled down the project, VS Code gives me some suggestions and I see now that I can't use the metadata keyword. However, I am not really able to fully have it set up locally, since the `helm dependency update` step fails with

```
Error: no repository definition for <MY-PATH>/saga-helm-chart/charts/app. Please add them via 'helm repo add'
Note that repositories must be URLs or aliases. For example, to refer to the "example"
repository, use "https://charts.example.com/" or "@example" instead of
"example". Don't forget to add the repo, too ('helm repo add').
```
Helm docs say:
> A chart repository is an HTTP server that houses an index.yaml file and optionally some packaged charts. When you're ready to share your charts, the preferred way to do so is by uploading them to a chart repository.

And I don't really know where the repository is in our case. I think this needs to be updated in the docs.

Reply:
okay I see now. I have tried running it locally using the documentation, however, it always fails on the 
`helm template . -f ./values.yaml -f ./environments/development.values.yaml --debug` step.
The error says:
```
Error: template: saga-product-service/charts/ap/templates/servicemonitor.yaml:4:11: executing "saga-product-service/charts/ap/templates/servicemonitor.yaml" at <include "nmp-chart.fullName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:42:32: executing "nmp-chart.fullName" at <include "nmp-chart.deploymentName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:15:22: executing "nmp-chart.deploymentName" at <include "nmp-chart.serviceName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:5:68: executing "nmp-chart.serviceName" at <63>: invalid value; expected string
helm.go:84: [debug] template: saga-product-service/charts/ap/templates/servicemonitor.yaml:4:11: executing "saga-product-service/charts/ap/templates/servicemonitor.yaml" at <include "nmp-chart.fullName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:42:32: executing "nmp-chart.fullName" at <include "nmp-chart.deploymentName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:15:22: executing "nmp-chart.deploymentName" at <include "nmp-chart.serviceName" .>: error calling include: template: saga-product-service/charts/ap/templates/_helpers.tpl:5:68: executing "nmp-chart.serviceName" at <63>: invalid value; expected string
```